certbot/cli.py

```diff
@@ -500,16 +499,23 @@ def prescan_for_flag(self, flag, possible_arguments):
             pass
         return True
 
-    def add(self, topic, *args, **kwargs):
+    def add(self, topics, *args, **kwargs):
         """Add a new command line argument.
 
-        :param str: help topic this should be listed under, can be None for
-                    "always documented"
+        :param topics: str or [str] help topic(s) this should be listed under,
+                       or None for "always documented"
```

Comment:
Question: This isn't new, but how is this supposed to work? It seems that the flags that are "always documented" only show up under all help output.